### PR TITLE
Error handling: Add generic boundary

### PR DESCRIPTION
**Pull request by andy-hook**
Add feedback for generic errors and when navigating to out of bounds proposalIds

image

### CODE REVIEW

File: src/components/Errors/GenericError.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                  >
+                    {detailsTitle}
+                  </h2>
+                )}
+                <p
+                  css={`
+                    color: ${theme.surfaceContentSecondary};
+                  `}
+                >
+                  {detailsContent}
```
Comment (delfipolito) [inline src/components/Errors/GenericError.js:115]: is detsailsContent always going to be present?

File: src/components/Errors/GenericError.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        </div>
+      )}
+      <div
+        css={`
+          display: grid;
+          grid-gap: ${1 * GU}px;
+          grid-template-columns: auto auto;
+        `}
+      >
+        <Button onClick={reportCallback}>Send Report</Button>
```
Comment (delfipolito) [inline src/components/Errors/GenericError.js:129]: Maybe we should make this function required in propTypes,  or render the button only if it is present

**Review by delfipolito — APPROVED** [review]
👏🏻👏🏻

File: src/components/Errors/GenericError.js (hunk 1 above)
Comment (andy-hook) [reply src/components/Errors/GenericError.js:115]: Yep, there'll always be a stack provided by an error.

File: src/components/Errors/GenericError.js (hunk 2 above)
Comment (andy-hook) [reply src/components/Errors/GenericError.js:129]: Good spot! this should only be rendered if provided.

**Pull request merged**